deployment/docker/docker-compose.yml

```diff
@@ -49,9 +49,9 @@ services:
       SCRUMLR_WEBSOCKET_URL: "${SCRUMLR_WEBSOCKET_URL}"
       SCRUMLR_SHOW_LEGAL_DOCUMENTS: "${SCRUMLR_SHOW_LEGAL_DOCUMENTS}"
       # Add missing frontend environment variables here
-      # SCRUMLR_LISTEN_PORT: "${SCRUMLR_LISTEN_PORT}"
-      # SCRUMLR_ANALYTICS_DATA_DOMAIN: "${ANALYTICS_DATA_DOMAIN}"
-      # SCRUMLR_ANALYTICS_SRC: "${ANALYTICS_SRC}"
+      SCRUMLR_LISTEN_PORT: "${SCRUMLR_LISTEN_PORT:-8080}"
```

Comment:
what does this mean?

Reply:
The <code>:-</code> defines a default value if the env var is not set. So if <code>SCRUMLR_LISTEN_PORT</code> is not set 8080 is used